src/AppContext.tsx

```diff
@@ -11,6 +11,7 @@ import {register as registerChatUser} from './chat/user';
 import {storage} from '@atb/storage';
 
 enum storeKey {
+  extendedOnboardingOnboarded = '@ATB_extended_onboarding_onboarded',
```

Comment:
Will this rerun the extended onboarding for those who have already seen it? Or is this just readding a previously removed storage value?

Reply:
It would have if it wasn't for [the second commit](https://github.com/AtB-AS/mittatb-app/pull/4275/commits/34d05b7f882c72e74b15a4f23a14bac0be582ea3).
It is a brand new storage key afaik, not sure if we have a list of previously used ones?